wazimap_ng/points/migrations/0052_auto_20220916_0733.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.28 on 2022-09-16 07:33
```

Comment:
If I add some custom things to a migration I like to change the name of the file but it's not needed just preference